src/ext/java/org/opentripplanner/ext/legacygraphqlapi/datafetchers/LegacyGraphQLQueryTypeImpl.java

```diff
@@ -736,30 +740,77 @@ public DataFetcher<DataFetcherResult<RoutingResponse>> plan() {
         "unpreferred.agencies",
         request.journey().transit()::setUnpreferredAgenciesFromString
       );
-      callWith.argument("banned.routes", request.journey().transit()::setBannedRoutesFromString);
-      callWith.argument("banned.agencies", request.journey().transit()::setBannedAgenciesFromSting);
-      callWith.argument("banned.trips", request.journey().transit()::setBannedTripsFromString);
+
       // callWith.argument("banned.stops", request::setBannedStops);
       // callWith.argument("banned.stopsHard", request::setBannedStopsHard);
       // callWith.argument("heuristicStepsPerMainStep", (Integer v) -> request.heuristicStepsPerMainStep = v);
       // callWith.argument("compactLegsByReversedSearch", (Boolean v) -> request.compactLegsByReversedSearch = v);
 
-      if (hasArgument(environment, "transportModes")) {
-        QualifiedModeSet modes = new QualifiedModeSet("WALK");
+      if (!hasArgument(environment, "banned") && !hasArgument(environment, "transportModes")) {
+        request.journey().transit().setFilters(List.of(AllowAllTransitFilter.of()));
```

Comment:
Again, should this not be somehow the default?

Reply:
List.of(AllowAllTransitFilter) is now a default. I removed this line of code.